spec/services/update_duplicate_matches_spec.rb

```diff
@@ -96,8 +86,8 @@
         expect(candidate2.reload.submission_blocked).to be(true)
       end
 
-      it 'sends an email to each candidate' do
-        expect { described_class.new.save! }.to change { ActionMailer::Base.deliveries.count }.by(2)
+      it 'sends only one email to each candidate in the match' do
```

Comment:
And should we specify that when the match is pre-existing, that only the newly added candidate is emailed?

Reply:
Done. ♻️